packages/react-charting/docs/TestPlans/LineChart/ComponentTests.md

```diff
@@ -0,0 +1,67 @@
+**Line Chart – Component test plan**
```

Comment:
Add test case for - gaps in line, events on line and colorFillBars.
The usage is there in our demo site.